pkgs/development/ruby-modules/gem/default.nix

```diff
@@ -184,6 +184,7 @@ stdenv.mkDerivation (attrs // {
       --install-dir "$GEM_HOME" \
       --build-root '/' \
       --backtrace \
+      --no-env-shebang \
```

Comment:
Good question. Most packages should either user the bundlerEnv or gemPackage which don't have these issues. But it's possible that some badly designed project could call `gem install`. I that case running `patchShebangs` in the postInstall would fix the issue but it's possible that this change could introduce a regression for those packages.